### DISCUSSION TITLE
KusionStack + KCL 22th Weekly Meeting | 第 22 次周会 (2023.09.14)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#7] [General]
会议方式
Peefy 邀请您参加腾讯会议
会议主题：KusionStack + KCL 社区双周会
会议时间：2023/08/03 20:00-20:45 (GMT+08:00) 中国标准时间 - 北京
重复周期：2023/08/03-2024/08/29 20:00-20:45, 每两周 (周四)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/Hc6sNpqTWnPb

#腾讯会议：778-2381-6338

复制该信息，打开手机腾讯会议即可参与

**Comment by SofiaPrescott** [comment]
Bedankt voor de uitnodiging. Ik heb de details en het programma van de bijeenkomst ontvangen. Het onderwerp klinkt interessant en ik ben blij dat de KusionStack- en KCL-community's regelmatig met elkaar in contact komen. De tweewekelijkse bijeenkomst past goed bij mij en het tijdslot is duidelijk. Ik ben van plan om de sessie op 3 augustus bij te [casino Betblast](https://betblastnl.nl/) wonen en, indien mogelijk, ook toekomstige bijeenkomsten te volgen. Laat het me weten als er van tevoren nog iets voorbereid moet worden of als er materiaal wordt gedeeld. Ik kijk uit naar de discussie en om van alle deelnemers te leren.
